media-video/baka-mplayer/baka-mplayer-2.0.4-r2.ebuild

```diff
@@ -1,17 +1,17 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-inherit qmake-utils
+inherit qmake-utils xdg-utils
```

Comment:
just xdg it?

EDIT: oh, I see you're syncing with live, nevermind unless feel like it

Reply:
It's displaying changes from the dropped -r0 to -r2, I didn't even notice it was there before ;p (not sync'ing anything)